src/app/georeferencer/qgsgeorefmainwindow.cpp

```diff
@@ -1322,19 +1328,19 @@ void QgsGeoreferencerMainWindow::saveGCPs()
   if ( pointFile.open( QIODevice::WriteOnly | QIODevice::Truncate ) )
   {
     QTextStream points( &pointFile );
+    points << QStringLiteral( "#CRS: %1" ).arg( mProjection.toWkt( QgsCoordinateReferenceSystem::WKT_PREFERRED ) );
     points << "mapX,mapY,pixelX,pixelY,enable,dX,dY,residual" << endl;
```

Comment:
The wkt needs a new line at the end. The file 'header' is appended to the wkt instead of being at a new line. This must also be adjusted when loading the file (read an extra line before parsing the gcps).

Reply:
Readline on line 1278 has been added to skip header.